### PR TITLE
[ORCA-483] Add github template for policy check command.

### PR DESCRIPTION
**Pull request by nishkrishnan**
This also fixes a a few bugs with ensuring versions, and resolving the local policy source.

### CODE REVIEW

File: server/events/markdown_renderer.go [hunk 1]
```diff
@@ -73,6 +74,11 @@ type planSuccessData struct {
 	PlanWasDeleted bool
 }
 
+type policyCheckSuccessData struct {
+	models.PolicyCheckSuccess
+	PlanWasDeleted bool
```
Comment (msarvar) [inline server/events/markdown_renderer.go:79]: Not sure if this is needed, policies are not ran if plans are being deleted. It is an explicit condition in the command_runner.

**Review by msarvar — COMMENTED** [review]
Looks good, let me know when you're done with tests I will approve the PR.

File: server/events/markdown_renderer.go (hunk 1 above)
Comment (nishkrishnan) [reply server/events/markdown_renderer.go:79]: What if we lose the ebs volume and the plan file gets deleted?

Comment (nishkrishnan) [reply server/events/markdown_renderer.go:79]: I guess I can remove it for now

**Review by msarvar — APPROVED** [review]

**Pull request merged**